tests/System/client_dms.sh

```diff
@@ -0,0 +1,127 @@
+echo " "
+echo " "
+echo " ########################## Data Management #############################"
+echo " "
+echo " ########################## BEGIN OF USER FILES TEST #############################"
+echo " "
+echo " "
+
+echo "====== dirac-proxy-init -g lhcb_user" #this is necesary to upload user files
+dirac-proxy-init -g lhcb_user
+
+dir=$( echo "$USER" |cut -c 1)/$USER
+echo "this is a test file" >> DMS_Scripts_Test_File.txt
+
+echo " "
+echo "====== dirac-dms-add-file /lhcb/user/$dir/Dirac_Scripts_Test_Directory/DMS_Scripts_Test_File.txt ./DMS_Scripts_Test_File.txt CERN-USER"
+dirac-dms-add-file /lhcb/user/$dir/Dirac_Scripts_Test_Directory/DMS_Scripts_Test_File.txt ./DMS_Scripts_Test_File.txt CERN-USER
+if [ $? -ne 0 ]
+then
+   exit $?
```

Comment:
I do not think we have to exit. Imagine if the CERN-USER is banned, we must continue testing, because it is not code related. In some test we use CNAF for testing, because CNAF is not in good shape I have to comment the exit in the code in order do not exit.